static/pages/home/home.html

```diff
@@ -58,6 +58,13 @@ <h3 class="md-headline oppia-foundation-headline">
       </div>
     </div>
   </md-card>
+<<<<<<< HEAD
```

Comment:
You miss these!

Reply:
Argh, sorry about that. Should be good now.